### DISCUSSION TITLE
Placeholder doesn't work in Custom N-N

### DISCUSSION THREAD

**Discussion by marcosp97** [khoait/DCE.PCF#97] [General]
![imagen_2024-02-16_081440629](https://github.com/khoait/DCE.PCF/assets/94583782/f134357b-1b20-41e5-975e-a369eedc5a6e)

![image](https://github.com/khoait/DCE.PCF/assets/94583782/965f11d5-e52e-4af4-b038-8e3c79fa0d8c)

**Comment by khoait** [comment]
Hi @marcosp97, thanks for reporting! I'll look into this soon.
